docs/flare_system_architecure.rst

```diff
@@ -0,0 +1,261 @@
+.. _flare_system_architecture:
+
+NVIDIA FLARE System Architecture
+=================================
+
+.. |flare_overview| image:: resources/flare_overview.png
+   :alt: FLARE Architecture Overview
+   :width: 45%
+
+.. |system_arch| image:: resources/system_architecture.png
+   :alt: FLARE Job Processing Architecture
+   :width: 45%
+
+|flare_overview| |system_arch|
+
+
+This document describes the overall system architecture of NVIDIA FLARE, including its layered structure, major subsystems,
+and how they interact. It covers the runtime components on both server and client sides, the communication framework,
+and the process model.
+
+The FLARE architecture (shown above) comprises three main layers:
+
+- **Foundation Layer** - Communication infrastructure, messaging protocols, privacy preservation tools, and secure platform management.
+- **Application Layer** - Building blocks for federated learning, including federation workflows and learning algorithms.
+- **Tooling** - FL Simulator and POC CLI for experimentation and simulation, plus deployment and management tools for production workflows.
+
+
+
+Core Components and Code Structure
+----------------------------------
+
+Primary System Modules
+######################
+
+Here's the simplified table without file paths:
+
+.. list-table:: **FLARE Core Components**
+   :header-rows: 1
+   :widths: 20 35 45
+
+   * - Component
+     - Primary Classes/Modules
+     - Purpose
+   * - FL Runtime
+     - ServerEngine, ClientEngine, JobRunner
+     - Core federated learning orchestration and execution
+   * - Job Management
+     - FedJob, JobDefManagerSpec, SimulatorRunner
+     - Job definition, storage, scheduling, and simulation
+   * - Communication
+     - Cell, CoreCell, StreamCell, Pipe
+     - Secure inter-party communication with streaming support
+   * - Client Integration
+     - ClientAPI (flare.receive(), flare.send()), LauncherExecutor
+     - ML framework integration and external process management
+   * - Administration
+     - FLAdminAPI, AdminCommandModule, Dashboard
+     - Programmatic and GUI-based system management
+   * - Deployment
+     - Lighter, ProvisionerSpec, WorkspaceBuilder
```

Comment:
Lighter? if we want to talk about Provision, 
Should be Participants, Builder and ... @IsaacYangSLA can comment more